### PR TITLE
Update country and language for Myanmar [Burma]

### PR DESCRIPTION
**Pull request by zarnipyo**

### CODE REVIEW

**Comment by z-ax** [comment]
Thanks for the PR! Let me check the change.

**Comment by zarnipyo** [comment]
First time OSS PR! Let me know if there are any issues.

**Comment by z-ax** [comment]
@zarnipyo cool, congrats on the first step!

Are you from the country? From what I've checked this name seems a proper one. There is also "Republic of the Union of Myanmar", but looks like it's still a subject to discussion.

Links:
- https://en.wikipedia.org/wiki/Names_of_Myanmar
- https://en.wikipedia.org/wiki/Myanmar

So I think we could merge it. Then I'll check the build and make a new version.

**Comment by zarnipyo** [comment]
Yes I was born and raised in Myanmar [Burma].

I'll give you some analogy. 
"Republic of the Union of Myanmar" (in native language ပြည်ထောင်စုသမ္မတ မြန်မာနိုင်ငံတော်) is the official name of the country Myanmar[Burma] (in native language မြန်မာ), being equivalent to "United Kingdom of Great Britain and Northern Ireland" for United Kingdom and "People's Republic of China" for China. I checked the name of the other countries and their native names. So I updated the native field to "မြန်မာ".

For language, there are over hundreds of languages in the country. But the official language of the country is Myanmasa (in native lanugage မြန်မာစာ). This language is originally spoken by the dominant ethnic group, Bamar (in native language ဗမာလူမျိုး). Thus sometimes people refer to the language as Bamarsa (in native language ဗမာစာ).

To sum up, **country: Myanmar[Burma] -> မြန်မာ. language: Myanmasa -> မြန်မာစာ**.

Let me know if you have any questions.

**Comment by z-ax** [comment]
Thank you for the details as well, from information I've got I agree with your changes.

By the way. Since this package was created I've been thinking about the list of **spoken** languages for each country. I think it might be pretty useful on some projects. If you'd share the list of spoken languages (I think it should contain languages for at least 10-15% of coverage) it would be pretty nice.

**Comment by z-ax** [comment]
Please check v2.4.0, it's published now.

**Comment by zarnipyo** [comment]
Thanks for the merge.

Diversity-wise, I think it's a good idea. For example, there are lots of Chinese dialects. But I think most of the internet users only care about national languages, the data in [languages.json](https://github.com/annexare/Countries/blob/master/data/languages.json). Population-wise national languages should covers more than 50%. So it depends on the use case. I, myself, can only list like 10 languages out of hundreds used in Myanmar. xD

**Comment by jbelien** [comment]
I think it's a bit strange and unexpected to have a `[` symbol in a country name.

```json
"MM": {
    "name": "Myanmar [Burma]",
    "native": "မြန်မာ",
    "phone": "95",
    "continent": "AS",
    "capital": "Naypyidaw",
    "currency": "MMK",
    "languages": [
      "my"
    ]
},
```

Maybe it would be better to add a `alternate` field with `Burma` (instead of `[Burma]` next to `Myanmar`) ?
What do you think ?

**Comment by zarnipyo** [comment]
I don't see any other countries having alternative names like this one. 
In my opinion, for JSON it's fine to include `alternative` field but for the library users I don't think they will use that field just for one country. 
For structured data like SQL, have the `alternative` column just for one country is not worth it.
I think may be it's time to deprecate the old name `Burma` (even though I prefer `Burma` for sentimental reasons).
How about just `Myanmar` instead of `Myanmar [Burma]`?

**Comment by jbelien** [comment]
> How about just Myanmar instead of Myanmar [Burma]?

I could definitely go with `Myanmar` only.
I proposed to add that `alternate` name because `Burma` seemed also important but I agree with you, it's probably not "worth" it if it is used only for one country.
That being said that `alternate` field could also be used for Belgium with "Kingdow of Belgium" or "Belgique", "België", and "Belgien" (official name in the 3 official languages).

**Comment by zarnipyo** [comment]
> That being said that `alternate` field could also be used for Belgium with "Kingdow of Belgium" or "Belgique", "België", and "Belgien" (official name in the 3 official languages).

If that's the case, go with the `alternative` field then.

**Comment by jbelien** [comment]
What do you think @z-ax ? :)

**Comment by z-ax** [comment]
I like the idea with the main name and array of `alternative` ones, seen some countries have that actually.

Let's continue this in a separate issue and agree on format: #35 @jbelien @zarnipyo. I think we could cover that with countries we could find alternative names, in a separate branch.
